### DISCUSSION TITLE
v23beta在wayland桌面模式下无法更换主题

### DISCUSSION THREAD

**Discussion by Linux-qitong** [linuxdeepin/developer-center#4399] [Troubleshooting | 故障排查] [answered]
v23beta在wayland桌面模式下无法更换主题，具体在“设置——个性化——主题",更换主题没变化
原贴:https://bbs.deepin.org/phone/zh/post/256631
我估计，他使用的nvidia显卡，而nvidia显卡对wayland支持不好，建议增加像gnome一样，装好n卡驱动直接给wayland闭了。

**Comment by Decodetalkers** [comment]
我想和N卡无关，N卡只是会影响图形显示，主题都没有变那应该是真的没有设置上

**Reply by qq4945286** [reply]
我用的amd显卡也是一样

**Comment by BLumia** [comment, marked as answer]
不是 n 卡问题，应该是 bug。

- 关联： https://github.com/linuxdeepin/developer-center/issues/4401

**Reply by Linux-qitong** [reply]
应该是BUG，我N卡dde用wayland直接黑屏进不去。

**Reply by Decodetalkers** [reply]
> 应该是BUG，我N卡dde用wayland直接黑屏进不去。

> N卡驱动390以下的是不支持Wayland的

**Reply by Linux-qitong** [reply]
我的是470驱动

**Reply by Decodetalkers** [reply]
> 我的是470驱动

我记错了，是495,你的N卡确实驱动不支持，你是什么架构的N卡，要升级下

**Reply by Linux-qitong** [reply]
升级不了，GTX650显卡，官方只提供了470驱动。
